modules/features/podcasts/src/main/java/au/com/shiftyjelly/pocketcasts/podcasts/view/podcast/PodcastHeader.kt

```diff
@@ -467,6 +489,82 @@ private fun PodcastActions(
         val dummyButtonWidthDp = dummyButton.width.toDp()
         val dummyButtonHeightDp = dummyButton.height.toDp()
 
+        val donateButton = subcompose("donateButton") {
+            val backgroundColor = Color.Transparent
+            val iconColor by transition.animateColor(
+                label = "donateButtonIconColor",
+                transitionSpec = { followColorSpec },
+                targetValueByState = { followed -> if (followed) MaterialTheme.theme.colors.primaryIcon03 else MaterialTheme.theme.colors.primaryIcon02 },
```

Comment:
According to Figma, I tried `primaryIcon02Active` for the unfollowed state icon color. It did not look correct in the extra dark theme type. I've changed it to  `primaryIcon02`. This also matches the [icon color in iOS](https://github.com/Automattic/pocket-casts-ios/blob/69b5d86acb6b455826e4ed7877a177b5051cca2c/podcasts/PodcastHeaderView.swift#L174). 

<img width="500" alt="Screenshot 2025-04-01 at 9 25 33 AM" src="https://github.com/user-attachments/assets/3718200c-bfa8-44e5-bbc1-6b7ff0f8e782" />